examples/00-load/create-polyhedron.py

```diff
@@ -0,0 +1,170 @@
+"""
+.. _polyhedron_example:
+
+Combining a polyhedron with other cells
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This example shows how to build a simple :class:`pyvista.UnstructuredGrid` using
+polyhedra. We will be using VTK types to determine which type of cells we are building.
+
+First, we import the required libraries.
+"""
+
+import numpy as np
+
+import pyvista as pv
+
+###############################################################################
+# Node arrays of the cells
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# We will mix several cells in one grid for this example. We set the points that
+# will define each cell.
+
+
+quad_points = [
+    [0.0, 0.0, 0.0],  # 0
+    [0.0, 0.01, 0.0],  # 1
+    [0.01, 0.01, 0.0],  # 2
+    [0.01, 0.0, 0.0],  # 3
+]
+polygon_points = [
+    [0.02, 0.0, 0.0],  # 4
+    [0.02, 0.01, 0.0],  # 5
+    [0.03, 0.01, 0.0],  # 6
+    [0.035, 0.005, 0.0],  # 7
+    [0.03, 0.0, 0.0],  # 8
+]
+hexa_points = [
+    [0.0, 0.0, 0.02],  # 9
+    [0.0, 0.01, 0.02],  # 10
+    [0.01, 0.01, 0.02],  # 11
+    [0.01, 0.0, 0.02],  # 12
+    [0.0, 0.0, 0.03],  # 13
+    [0.0, 0.01, 0.03],  # 14
+    [0.01, 0.01, 0.03],  # 15
+    [0.01, 0.0, 0.03],  # 16
+]
+polyhedron_points = [
+    [0.02, 0.0, 0.02],  # 17
+    [0.02, 0.01, 0.02],  # 18
+    [0.03, 0.01, 0.02],  # 19
+    [0.035, 0.005, 0.02],  # 20
+    [0.03, 0.0, 0.02],  # 21
+    [0.02, 0.0, 0.03],  # 22
+    [0.02, 0.01, 0.03],  # 23
+    [0.03, 0.01, 0.03],  # 24
+    [0.035, 0.005, 0.03],  # 25
+    [0.03, 0.0, 0.03],  # 26
+]
+points = np.array(quad_points + polygon_points + hexa_points + polyhedron_points)
+
+
+###############################################################################
+# Connectivity arrays
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# We set which points each of the cells needs. The first element in each array
+# is the number of elements the cell will have. E.g., quad array is composed of
+# 0, 1, 2 and 3 points, so it has 4 elements. This is needed because the connectivity
+# array that will contain all cells needs to have only one dimension, so to be able to
+# know which points belong to which cell, we set that the following N positions in the array
+# belong to a single cell, the next N positions belong to another cell, etc.
+
+quad = np.array([4, 0, 1, 2, 3])
+polygon = np.array([5, 4, 5, 6, 7, 8])
+hexa = np.array([8, 9, 10, 11, 12, 13, 14, 15, 16])
+
+###############################################################################
+# Polyhedron connectivity array
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Polyhedrons connectivity array is defined differently from the rest of the cell types.
+# For polyhedrons, we need to set the faces with the following format:
```

Comment:
Since you accepted my first "polyhedra" suggestion:
```suggestion
# The connectivity array of polyhedra is defined differently from the rest of the cell
# types. For polyhedra, we need to set the faces with the following format:
```